private/webpack/dev.js

```diff
@@ -7,14 +6,8 @@ const h = require('./helper');
 const cwd = process.cwd();
 
 module.exports = merge(base, {
-  devtool: 'cheap-module-eval-source-map',
-  devServer: {
-    contentBase: path.resolve(__dirname, 'app'),
-    port: 1234
```

Comment:
webpack-dev-server now figures this out and automatically opens the page for ya: https://github.com/seedom-io/seedom-web3/pull/17/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R61

As an extra bonus, you can in theory run multiple dev servers as it just does a +1 on the port number if not available.